index.bs

```diff
@@ -563,75 +568,125 @@ This method takes the following parameters:
-        operation on that authenticator and remove its entry from the list.
-    - If any authenticator returns a status indicating that the user cancelled the operation, delete that authenticator's entry
-        from |issuedRequests|. For each remaining entry in |issuedRequests| invoke the [=authenticatorCancel=] operation on
-        that authenticator, and remove its entry from the list.
-    - If any authenticator returns an error status, delete the corresponding entry from |issuedRequests|.
-    - If any authenticator returns success:
-        - Remove this authenticator's entry from |issuedRequests|.
-        - Create a new {{AuthenticationAssertion}} object named |value| and populate its fields with the values returned from the
-            authenticator and |clientDataJSON|.
-        - For each remaining entry in |issuedRequests| invoke the [=authenticatorCancel=] operation on that authenticator and
-            remove its entry from the list.
-        - Resolve |promise| with |value| and terminate this algorithm.
-
-1. Reject |promise| with a {{DOMException}} whose name is "{{NotAllowedError}}", and terminate this algorithm.
+1. For each |authenticator| currently available on this platform, perform the following steps:
+
+    1. Let |credentialList| be a new [=list=].
+
+    1. If <code>{{options}}.{{AssertionOptions/allowList}}</code> [=list/is not empty=], execute a 
+        platform-specific procedure to determine which, if any, credentials in
+        <code>{{options}}.{{AssertionOptions/allowList}}</code> are present on this |authenticator| by matching with 
+        <code>{{options}}.{{AssertionOptions/allowList}}.{{ScopedCredentialDesciptor/id}}</code> and 
+        <code>{{options}}.{{AssertionOptions/allowList}}.{{ScopedCredentialDesciptor/type}}</code>, and set |credentialList| to
+        this filtered list.
+    
+    1. If |credentialList| [=list/is empty=] then [=continue=].
+
+    1. [=In parallel=], [=list/for each=] credential |C| in |credentialList|:
+        1. If <code>|C|.{{transports}}</code> [=list/is not empty=], the client SHOULD select one |transport| from
+            {{transports}}. Then, using |transport|, invoke the [=authenticatorGetAssertion=] operation on |authenticator|, with
+            |rpId|, |clientDataHash|, |credentialList|, and |clientExtensions| as parameters.
+        1. Otherwise, using local configuration knowledge of the appropriate transport to use with |authenticator|, invoke the 
+            [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|, |clientDataHash|, |credentialList|, and
+            |clientExtensions| as parameters.
+    
+    1. [=set/Append=] |authenticator| to |issuedRequests|.
+
+1. Let |promise| be [=a new Promise=]. Return |promise| and start a timer for |adjustedTimeout| milliseconds. Then execute the
+    following steps [=in parallel=].
```

Comment:
Should this line end in a colon, then, since it's sort-of starting another block?

By that logic, maybe we should up the indentation level, too?